app/configuration.rb

```diff
@@ -27,8 +27,9 @@ module GitlabMonitor
     #Implementation of class responsible for showing notifications in system. Specify hide timeout [s].
     #You can use equations like 60*5 -> 5 minutes
     config.notifier          = (RbConfig::CONFIG['host_os'].match(/mswin|windows/i) \
-                                  ? WindowsNotificationExecutor 
+                                  ? WindowsNotificationExecutor
                                   : LinuxNotificationExecutor).new(time: 5)
+    config.link_enabled        = true
```

Comment:
Dodałeś pozycję do konfiga aplikacji, a użyta jest tylko w jednej implementacji egzekutora notyfikacji. Wolałbym raczej żeby był to parametr LinuxNotificationExecutor'a który i tak przyjmuje już hash'a z opcjami